packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -372,6 +371,40 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
         );
     };
 
+    const getLanguageRoutesTraderHub = () => {
```

Comment:
@yauheni-deriv Let's make it a component and wrap it with `useCallback` instead of function.